open-company/open-company-bot
# Eastwood config

### Review after #55 

To test:
- start the bot
- [x] on localhost run `lein eastwood` and make sure that there are no errors
- [x] share to slack
- [x] send yourself a Slack digest
- [x] invite a user via Slack (using a test org?)
- make sure you have mention and notifications via Slack
- [x] mention the Slack user with another user
- [x] add a post and w/ another user comment on the post so you get a notification via Slack
- [x] send a message to the bot and check that it replies with the usage message
- run any other Slack test you can think of
